src/didactWebView.ts

```diff
@@ -87,7 +87,11 @@ export class DidactWebviewPanel {
 	public static hardReset() {
 		if (DidactWebviewPanel.currentPanel) {
 			DidactWebviewPanel.currentPanel.setDidactStr(undefined);
-			DidactWebviewPanel.currentPanel.setDidactUriPath(undefined);
+			const configuredUri : string | undefined = vscode.workspace.getConfiguration().get('didact.defaultUrl');
```

Comment:
using the constant will be more robust https://github.com/redhat-developer/vscode-didact/blob/c4c1f342d6cd1a5235288dcbbbecc2afa625ae80/src/utils.ts#L24